src/main/java/com/fasttime/domain/resume/dto/ResumeResponseDto.java

```diff
@@ -0,0 +1,15 @@
+package com.fasttime.domain.resume.dto;
+
+import lombok.Builder;
+
+@Builder
+public record ResumeResponseDto(
+        Long id,
+        String title,
+        String content,
+        String writer,
+        int likeCount,
+
+        int viewCount) {
```

Comment:
```suggestion
        int viewCount
) {
```
수정해주시면 좀 더 일관성 있는 코드가 될 것 같습니다~